src/controllers/handlers/scene-admin-handlers/remove-scene-admin-handler.ts

```diff
@@ -0,0 +1,74 @@
+import { EthAddress } from '@dcl/schemas'
+import { HandlerContextWithPath, InvalidRequestError, UnauthorizedError } from '../../../types'
+import { validate } from '../utils'
+
+export async function removeSceneAdminHandler(
+  ctx: Pick<
+    HandlerContextWithPath<'sceneAdminManager' | 'sceneFetcher' | 'logs' | 'config' | 'fetch', '/scene-admin'>,
+    'components' | 'url' | 'params' | 'verification' | 'request'
+  >
+) {
+  const {
+    components: { logs, sceneFetcher, sceneAdminManager },
+    request,
+    verification
+  } = ctx
+
+  const logger = logs.getLogger('remove-scene-admin-handler')
+
+  const { getPlace, hasLandPermission, hasWorldPermission } = sceneFetcher
+
+  if (!verification?.auth) {
+    throw new UnauthorizedError('Authentication required')
+  }
+
+  const payload = await request.json()
+
+  if (!payload.admin) {
+    logger.warn(`Invalid scene admin payload`, payload)
+    throw new InvalidRequestError(`Invalid payload`)
+  }
+
+  const { parcel, hostname, realmName } = await validate(ctx)
+  const isWorlds = hostname.includes('worlds-content-server')
+  const authAddress = verification.auth.toLowerCase()
+
+  if (!EthAddress.validate(payload.admin)) {
+    throw new UnauthorizedError('Invalid admin address')
+  }
+
+  const place = await getPlace(isWorlds, realmName, parcel)
+  if (!place) {
+    throw new InvalidRequestError('Place not found')
+  }
+
+  const hasPermission =
+    (isWorlds
+      ? await hasWorldPermission(authAddress, place.world_name!)
+      : await hasLandPermission(authAddress, place.positions)) ||
+    (await sceneAdminManager.isAdmin(place.id, authAddress))
```

Comment:
What's the difference between `hasLandPermission` and `isAdmin`, shouldn't an admin return true for `hasLandPermission`?

Reply:
hasLandPermission is asking to catalyst of the address has permissions to the land. is Admin, is an admin to stream in the scene